build/all-confirmed.txt

```diff
@@ -124,7 +124,6 @@
 99\,\ 108\,\ 111\,\ 117\,\ 100\,\ 115\,\ 101\,\ 114\,\ 118\,\ 105\,\ 99\,\ 101\,\ 46\,\ 116\,\ 119
 99\,\ 108\,\ 111\,\ 117\,\ 100\,\ 45\,\ 117\,\ 112\,\ 100\,\ 97\,\ 116\,\ 101\,\ 46\,\ 116\,\ 111\,\ 112
 <Files [^>]+.(jpg|png|gif)>\s*ForceType application\/x-httpd-php
-<\?php\s+eval\(base64_decode\(
 <\?php\s+eval\(gzinflate\(
```

Comment:
Yes, however, no FPs on our platform. Yours? We got to find the thin line between malware and legitimate code, and as you can see that work is mostly empirical :)